src/lib/dns-server/service_widget_helpers.rb

```diff
@@ -0,0 +1,22 @@
+require "cwm/service_widget"
+require "yast2/system_service"
+
+module Y2DnsServer
+  module ServiceWidgetHelpers
```

Comment:
NP: Do you think this "helper" pays off? At the end, it only contains to simple methods.

Reply:
The other option is to have this two simple methods duplicated, reason why I move them here.